crates/iota-indexer/src/ingestion/primary/prepare.rs

```diff
@@ -368,12 +368,21 @@ impl PrimaryWorker {
         checkpoint_timestamp_ms: u64,
         metrics: &IndexerMetrics,
     ) -> IndexerResult<IndexedTransactionComponents> {
+        let db_txn = try_new_indexed_transaction(
+            tx,
+            tx_sequence_number,
+            checkpoint_seq,
+            checkpoint_timestamp_ms,
+            metrics,
+        )
+        .await?;
+
         let CheckpointTransaction {
             transaction: sender_signed_data,
             effects: fx,
             events,
-            input_objects,
-            output_objects,
+            input_objects: _,
+            output_objects: _,
```

Comment:
[Nit](https://doc.rust-lang.org/book/ch19-03-pattern-syntax.html#remaining-parts-of-a-value-with-)
```suggestion
            ..
```